framework/test/gtest/MultipleListenersTest.cpp

```diff
@@ -246,7 +246,7 @@ namespace
       public:
         MultipleListenersTest() : framework(FrameworkFactory().NewFramework()) {}
 
-        ~MultipleListenersTest() override = default;
+        ~MultipleListenersTest() override { count = 0; }
```

Comment:
Curious as to the reason for this change.

Reply:
@carneyweb count was a variable declared in the namespace that, when run using `gtest_repeat` was not getting cleared/reset after each run and so we were erroring because it was a running sum as opposed to a sum being reset each time. For that reason, I am using the destructor for the test to reset that count variable for each iteration of the test

Reply:
I'd suggest a comment that says exactly that :-)